core/ErrorHandler.php

```diff
@@ -60,7 +61,20 @@ public static function getFatalErrorPartialBacktrace()
                 $function = $entry['class'] . $entry['type'] . $function;
             }
 
-            $result .= sprintf("#%s %s(%s): %s()\n", $index, $entry['file'], $entry['line'], $function);
+            $args = '';
+            if (!empty($entry['args'])) {
+                $isFirst = true;
+                foreach ($entry['args'] as $name => $value) {
+                    if ($isFirst) {
+                        $isFirst = false;
+                    } else {
+                        $args .= ', ';
+                    }
+                    $args .= $name . '=' . $value;
```

Comment:
just fyi: Was wondering if we need to escape something here but by the looks not needed as it is printed with `{{ lastError.message|e('url') }}`